packages/web/server/queries/complex/assets/index.ts

```diff
@@ -86,26 +84,19 @@ export async function getAssets({
 }
 
 /**
- * This function maps raw assets to a CoinPretty. This is useful for
+ * This function coins to a CoinPretty if listed in asset list. This is useful for
  * converting raw assets returned from chain into coins listed in asset list.
- * It also optionally calculates the fiat value of the asset if the 'calculatePrice'
- * parameter is true.
  *
  * @param rawAssets An array of raw assets. Each raw asset is an object with an 'amount' and 'denom' property.
- * @param calculatePrice A boolean indicating whether to calculate the price of the asset.
  *
- * @returns A promise that resolves to an array of CoinPretty objects. Each CoinPretty object represents an asset and has an optional 'fiatValue' property.
+ * @returns A promise that resolves to an array of CoinPretty objects. Each CoinPretty object represents an asset that is listed. Unlisted assets are filtered.
  */
-export async function mapAssetsToCoins({
-  rawAssets,
-  calculatePrice,
-}: {
-  rawAssets?: {
+export async function mapListedCoins(
```

Comment:
What do you think if we rename this function to something more specific?
```suggestion
export async function mapRawCoinToPretty(
```